<launch>
  <node pkg="hdr_attr_ctrl" 
        type="acquisition" 
        name="acquisition" 
        output="screen" 
        args="$(find hdr_attr_ctrl)/param/quad_cameras.yaml"/>
</launch>